### ISSUE TITLE
Could not open any SVG in Part Editor

### ISSUE THREAD

**Issue by exocode** [fritzing/fritzing-app#3188]
I wann add a 7-segment-display part, by editing the original one and save it with a different name.

When I open a new image, the fritzing app gives me an error, but in the most cases it crashes.

Am I doing something wrong?

best regards

Screenrecord:

![2016-02-08 at 16 56](https://cloud.githubusercontent.com/assets/1597621/12890702/0d38f9b2-ce85-11e5-811c-93d715c690ca.gif)

my tried svg:

```

<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1 Basic//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11-basic.dtd">
<svg version="1.1" baseProfile="basic" id="Layer_1" xmlns:svg="http://www.w3.org/2000/svg"
     xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" viewBox="0 0 114.3 38.9"
     xml:space="preserve">
<g>
    <g id="breadboard">
        <circle id="connector0pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="31.9" cy="33.8" r="2"/>
        <g id="connector0terminal">
        </g>
        <path fill="#9A916C" d="M3.4,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C0.9,29.9,2,28.8,3.4,28.8L3.4,28.8z M2,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C2.6,32.4,2,33,2,33.8C2,33.8,2,33.8,2,33.8z"/>
        <circle id="connector1pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="39.1" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M10.6,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C8.1,29.9,9.2,28.8,10.6,28.8L10.6,28.8z M9.2,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C9.8,32.4,9.2,33,9.2,33.8C9.2,33.8,9.2,33.8,9.2,33.8z"/>
        <circle id="connector2pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="46.3" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M17.8,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C15.3,29.9,16.4,28.8,17.8,28.8L17.8,28.8z M16.4,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            S16.4,33,16.4,33.8C16.4,33.8,16.4,33.8,16.4,33.8z"/>
        <circle id="connector3pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="53.5" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M25,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C22.5,29.9,23.6,28.8,25,28.8L25,28.8z M23.6,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            S23.6,33,23.6,33.8L23.6,33.8z"/>
        <circle id="connector4pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="60.7" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M32.2,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C29.7,29.9,30.8,28.8,32.2,28.8L32.2,28.8z M30.8,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            C31.4,32.4,30.8,33,30.8,33.8L30.8,33.8z"/>
        <circle id="connector5pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="67.9" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M39.4,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C36.9,29.9,38,28.8,39.4,28.8L39.4,28.8z M38,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            C38.6,32.4,38,33,38,33.8L38,33.8z"/>
        <circle id="connector7pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="82.3" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M53.8,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C51.3,29.9,52.4,28.8,53.8,28.8L53.8,28.8z M52.4,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S52.4,33,52.4,33.8
            L52.4,33.8z"/>
        <circle id="connector6pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="75.1" cy="33.8" r="2"/>
        <path fill="#9A916C" d="M46.6,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C44.1,29.9,45.2,28.8,46.6,28.8L46.6,28.8z M45.2,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S45.2,33,45.2,33.8
            L45.2,33.8z"/>
        <path fill="#9A916C" d="M61.3,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C58.8,29.9,59.9,28.8,61.3,28.8L61.3,28.8z M59.9,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4c0,0,0,0,0,0
            c0-0.8-0.6-1.4-1.4-1.4S59.9,33,59.9,33.8C59.9,33.8,59.9,33.8,59.9,33.8z"/>
        <path fill="#9A916C" d="M68.5,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C66,29.9,67.1,28.8,68.5,28.8L68.5,28.8z M67.1,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C67.7,32.4,67.1,33,67.1,33.8C67.1,33.8,67.1,33.8,67.1,33.8z"/>
        <path fill="#9A916C" d="M75.7,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C73.2,29.9,74.3,28.8,75.7,28.8L75.7,28.8z M74.3,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            S74.3,33,74.3,33.8C74.3,33.8,74.3,33.8,74.3,33.8z"/>
        <path fill="#9A916C" d="M82.9,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C80.4,29.9,81.5,28.8,82.9,28.8L82.9,28.8z M81.5,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            S81.5,33,81.5,33.8L81.5,33.8z"/>
        <path fill="#9A916C" d="M90.1,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C87.6,29.9,88.7,28.8,90.1,28.8L90.1,28.8z M88.7,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            C89.3,32.4,88.7,33,88.7,33.8L88.7,33.8z"/>
        <path fill="#9A916C" d="M97.3,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C94.8,29.9,95.9,28.8,97.3,28.8L97.3,28.8z M95.9,33.8c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            C96.5,32.4,95.9,33,95.9,33.8L95.9,33.8z"/>
        <path fill="#9A916C" d="M111.7,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C109.2,29.9,110.3,28.8,111.7,28.8L111.7,28.8z M110.3,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            S110.3,33,110.3,33.8L110.3,33.8z"/>
        <path fill="#9A916C" d="M104.5,28.8c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C102,29.9,103.1,28.8,104.5,28.8L104.5,28.8z M103.1,33.8c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4
            S103.1,33,103.1,33.8L103.1,33.8z"/>
        <circle id="connector8pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="82.3" cy="5" r="2"/>
        <path fill="#9A916C" d="M53.9,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5l0-5
            C51.4,1.1,52.5,0,53.9,0L53.9,0z M52.5,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S52.5,4.2,52.5,5L52.5,5z"/>
        <circle id="connector13pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="46.3" cy="5" r="2"/>
        <path fill="#9A916C" d="M17.9,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C15.4,1.1,16.5,0,17.9,0L17.9,0z M16.5,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            S16.5,4.2,16.5,5C16.5,5,16.5,5,16.5,5z"/>
        <circle id="connector12pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="53.5" cy="5" r="2"/>
        <path fill="#9A916C" d="M25.1,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C22.6,1.1,23.7,0,25.1,0L25.1,0z M23.7,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S23.7,4.2,23.7,5L23.7,5z"
            />
        <circle id="connector11pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="60.7" cy="5" r="2"/>
        <path fill="#9A916C" d="M32.3,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C29.8,1.1,30.9,0,32.3,0L32.3,0z M30.9,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4C31.5,3.6,30.9,4.2,30.9,5
            L30.9,5z"/>
        <circle id="connector9pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="75.1" cy="5" r="2"/>
        <path fill="#9A916C" d="M46.7,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C44.2,1.1,45.3,0,46.7,0L46.7,0z M45.3,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S45.3,4.2,45.3,5L45.3,5z"/>
        <circle id="connector10pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="67.9" cy="5" r="2"/>
        <path fill="#9A916C" d="M39.5,0C40.9,0,42,1.1,42,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0C38.1,10.1,37,9,37,7.6l0-5
            C37,1.1,38.1,0,39.5,0L39.5,0z M38.1,5c0,0.8,0.6,1.4,1.4,1.4C40.3,6.5,41,5.8,41,5s-0.6-1.4-1.4-1.4C38.7,3.6,38.1,4.2,38.1,5
            L38.1,5z"/>
        <circle id="connector14pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="39.1" cy="5" r="2"/>
        <path fill="#9A916C" d="M10.7,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5l0-5
            C8.2,1.1,9.3,0,10.7,0L10.7,0z M9.3,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C9.9,3.6,9.3,4.2,9.3,5C9.3,5,9.3,5,9.3,5z"/>
        <circle id="connector15pin" fill="none" stroke="#9A916C" stroke-width="0.381" cx="31.9" cy="5" r="2"/>
        <path fill="#9A916C" d="M3.5,0C4.9,0,6,1.1,6,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0C2.1,10.1,1,9,1,7.6l0-5C1,1.1,2.1,0,3.5,0L3.5,0z
             M2.1,5c0,0.8,0.6,1.4,1.4,1.4C4.3,6.5,5,5.8,5,5c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4C2.7,3.6,2.1,4.2,2.1,5C2.1,5,2.1,5,2.1,5z"/>
        <path fill="#9A916C" d="M111.7,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5l0-5
            C109.2,1.1,110.3,0,111.7,0L111.7,0z M110.3,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S110.3,4.2,110.3,5L110.3,5
            z"/>
        <path fill="#9A916C" d="M75.7,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C73.2,1.1,74.3,0,75.7,0L75.7,0z M74.3,5c0,0.8,0.6,1.4,1.4,1.4s1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            S74.3,4.2,74.3,5C74.3,5,74.3,5,74.3,5z"/>
        <path fill="#9A916C" d="M82.9,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C80.4,1.1,81.5,0,82.9,0L82.9,0z M81.5,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4S81.5,4.2,81.5,5L81.5,5z"
            />
        <path fill="#9A916C" d="M90.1,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C87.6,1.1,88.7,0,90.1,0L90.1,0z M88.7,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4C89.3,3.6,88.7,4.2,88.7,5
            L88.7,5z"/>
        <path fill="#9A916C" d="M104.5,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5v-5
            C102,1.1,103.1,0,104.5,0L104.5,0z M103.1,5c0,0.8,0.6,1.4,1.4,1.4S106,5.8,106,5s-0.6-1.4-1.4-1.4S103.1,4.2,103.1,5L103.1,5z"/>
        <path fill="#9A916C" d="M97.3,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5l0-5
            C94.8,1.1,95.9,0,97.3,0L97.3,0z M95.9,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4s-0.6-1.4-1.4-1.4C96.5,3.6,95.9,4.2,95.9,5
            L95.9,5z"/>
        <path fill="#9A916C" d="M68.5,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0C67.1,10.1,66,9,66,7.6l0-5
            C66,1.1,67.1,0,68.5,0L68.5,0z M67.1,5c0,0.8,0.6,1.4,1.4,1.4S70,5.8,70,5c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C67.7,3.6,67.1,4.2,67.1,5C67.1,5,67.1,5,67.1,5z"/>
        <path fill="#9A916C" d="M61.3,0c1.4,0,2.5,1.1,2.5,2.5v5c0,1.4-1.1,2.5-2.5,2.5l0,0c-1.4,0-2.5-1.1-2.5-2.5l0-5
            C58.8,1.1,59.9,0,61.3,0L61.3,0z M59.9,5c0,0.8,0.6,1.4,1.4,1.4c0.8,0,1.4-0.6,1.4-1.4c0,0,0,0,0,0c0-0.8-0.6-1.4-1.4-1.4
            C60.5,3.6,59.9,4.2,59.9,5C59.9,5,59.9,5,59.9,5z"/>
        <polygon fill="#323232" points="-0.1,1 113.3,1 114.4,2 114.4,37.9 0.9,37.9 -0.1,36.6        "/>
        <polygon fill="#646464" points="9.6,4.5 22,4.5 22.9,5.3 21.2,7.8 10.4,7.8 8.8,5.3       "/>
        <polygon fill="#EF1818" points="23.7,6.2 24.5,7.8 22.9,16.9 21.2,18.6 19.6,16.9 21.2,8.6        "/>
        <polygon fill="#EF1818" points="21.2,19.4 22,21.1 20.4,30.2 18.7,31.9 17.1,30.2 18.7,21.9       "/>
        <polygon fill="#646464" points="17.1,33.5 4.6,33.5 3.8,32.7 5.5,30.2 16.2,30.2 17.9,32.7        "/>
        <polygon fill="#EF1818" points="3,31.9 2.1,30.2 3.8,21.1 5.5,19.4 7.1,21.1 5.5,29.4         "/>
        <polygon fill="#EF1818" points="5.5,18.6 4.6,16.9 6.3,7.8 7.9,6.2 9.6,7.8 7.9,16.1      "/>
        <polygon fill="#EF1818" points="6.3,18.6 7.9,16.9 18.7,16.9 20.4,18.6 18.7,20.3 7.9,20.3        "/>
        <circle fill="#646464" cx="22.5" cy="31.9" r="1.7"/>
        <polygon fill="#646464" points="38.4,4.5 50.8,4.5 51.7,5.3 50,7.8 39.2,7.8 37.6,5.3         "/>
        <polygon fill="#EF1818" points="52.5,6.2 53.3,7.8 51.7,16.9 50,18.6 48.3,16.9 50,8.6        "/>
        <polygon fill="#EF1818" points="50,19.4 50.8,21.1 49.2,30.2 47.5,31.9 45.9,30.2 47.5,21.9       "/>
        <polygon fill="#EF1818" points="45.9,33.5 33.4,33.5 32.6,32.7 34.2,30.2 45,30.2 46.7,32.7       "/>
        <polygon fill="#EF1818" points="31.8,31.9 30.9,30.2 32.6,21.1 34.2,19.4 35.9,21.1 34.2,29.4         "/>
        <polygon fill="#EF1818" points="34.2,18.6 33.4,16.9 35.1,7.8 36.7,6.2 38.4,7.8 36.7,16.1        "/>
        <polygon fill="#646464" points="35.1,18.6 36.7,16.9 47.5,16.9 49.2,18.6 47.5,20.3 36.7,20.3         "/>
        <circle fill="#646464" cx="51.3" cy="31.9" r="1.7"/>
        <circle fill="#646464" cx="55.5" cy="25.3" r="1.7"/>
        <circle fill="#646464" cx="57.5" cy="12.4" r="1.7"/>
        <circle fill="#646464" cx="87.6" cy="6" r="1.7"/>
        <polygon fill="#646464" points="67.2,4.5 79.6,4.5 80.5,5.3 78.8,7.8 68,7.8 66.4,5.3         "/>
        <polygon fill="#EF1818" points="81.3,6.2 82.1,7.8 80.5,16.9 78.8,18.6 77.1,16.9 78.8,8.6        "/>
        <polygon fill="#EF1818" points="78.8,19.4 79.6,21.1 78,30.2 76.3,31.9 74.7,30.2 76.3,21.9       "/>
        <polygon fill="#646464" points="74.7,33.5 62.2,33.5 61.4,32.7 63,30.2 73.8,30.2 75.5,32.7       "/>
        <polygon fill="#EF1818" points="60.6,31.9 59.7,30.2 61.4,21.1 63,19.4 64.7,21.1 63,29.4         "/>
        <polygon fill="#EF1818" points="63,18.6 62.2,16.9 63.9,7.8 65.5,6.2 67.2,7.8 65.5,16.1      "/>
        <polygon fill="#EF1818" points="63.9,18.6 65.5,16.9 76.3,16.9 78,18.6 76.3,20.3 65.5,20.3       "/>
        <circle fill="#646464" cx="80.1" cy="31.9" r="1.7"/>
        <polygon fill="#646464" points="96.1,4.5 108.5,4.5 109.3,5.3 107.7,7.8 96.9,7.8 95.2,5.3        "/>
        <polygon fill="#EF1818" points="110.2,6.2 111,7.8 109.3,16.9 107.7,18.6 106,16.9 107.7,8.6      "/>
        <polygon fill="#EF1818" points="107.7,19.4 108.5,21.1 106.9,30.2 105.2,31.9 103.5,30.2 105.2,21.9       "/>
        <polygon fill="#EF1818" points="103.5,33.5 91.1,33.5 90.3,32.7 91.9,30.2 102.7,30.2 104.4,32.7      "/>
        <polygon fill="#EF1818" points="89.4,31.9 88.6,30.2 90.3,21.1 91.9,19.4 93.6,21.1 91.9,29.4         "/>
        <polygon fill="#EF1818" points="91.9,18.6 91.1,16.9 92.8,7.8 94.4,6.2 96.1,7.8 94.4,16.1        "/>
        <polygon fill="#646464" points="92.8,18.6 94.4,16.9 105.2,16.9 106.9,18.6 105.2,20.3 94.4,20.3      "/>
        <circle fill="#646464" cx="109" cy="31.9" r="1.7"/>
        <polygon fill="#C7C7C7" points="113.4,36.6 -0.1,36.6 0.9,37.9 114.4,37.9        "/>
        <polygon fill="#AAAAAA" points="113.3,36.6 114.4,37.9 114.4,2 113.3,1       "/>
    </g>
</g>
</svg>

```

**Comment by nateevans** [comment]
@exocode - Know this has been a long time... but after banging my head on this with Illustrator SVGs here's what I found:

The main issue is that in the main `<svg>` tag, there is no `width` or `height` property set. You _could_ manually add those and save the SVG and it should import, but I found the easiest way is to get the SVG Options in Illustrator set correctly.

These settings work for me (set CSS Properties to "Presentation Attributes" and make sure to uncheck "Responsive". After saving like this, I could import to Fritzing no problem.):

![screen shot 2019-01-23 at 3 35 06 pm](https://user-images.githubusercontent.com/1561006/51635532-fb7bf780-1f24-11e9-94d6-eaa69ee02796.png)
